backend/package.json

```diff
@@ -1,41 +1,32 @@
 {
-  "name": "@ifixit/commerce-backend",
+  "name": "backend",
   "private": true,
-  "version": "1.0.0",
-  "description": "Strapi backend for iFixit e-commerce",
+  "version": "0.1.0",
```

Comment:
Why was the version dropped from 1.0.0 to 0.1.0?

Reply:
@ardelato don't review this code yet, wait for #158 to be merge, as this is a change that comes from that branch. Answering your question, I think it's just Strapi cli defaults, we aren't really using `version` numbers for anything